src/common/crc32c_s390x.c

```diff
@@ -1,14 +1,29 @@
 /*
- * CRC-32 algorithm implemented with the z/Architecture
- * Vector Extension Facility.
+ * CRC-32 algorithm implemented with the z/Architecture Vector Extension
+ * Facility.
```

Comment:
Nit: re-wrapping here and in other files seems unnecessary.

Reply:
I'll leave this as a nit. I had to add license notices so the header needed changed anyway.